index.html

```diff
@@ -42,6 +42,219 @@ <h2 class="connect-link">LET’S CONNECT</h2>
                 </li>
             </ul>
         </section>
+        <section class="card">
+            <article class="card-section">
+                <img src="./images/tonic-img-1.png" alt="tonic-snapshot" class="tonic-img">
+                <article class="card-section-text">
+                    <h2 class="heading-title">
+                        Tonic
+                    </h2>
+                    <article class="card-sub-container">
+                        <h2 class="title-canopy">CANOPY</h2>
+                        <img src="" alt="">
+                        <h2 class="title-developer">Back End Dev</h2>
+                        <img src="" alt="">
+                        <h2 class="title-year">2015</h2>
+                    </article>
+
+                    <p class="desciption-p-mobile">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="pro-language">
+                        <li>
+                            <h1 class="html-1">html</h1>
+                        </li>
+                        <li>
+                            <h1 class="html-1">css</h1>
+                        </li>
+                        <li>
+                            <h1 class="html-1">Javascript</h1>
+                        </li>
+                    </ul>
+                    <div class="btn-container">
+                        <button class="btn-see-project" type="button">See Project</button>
+                    </div>
+                </article>
+            </article>
+            <article class="card-section">
+                <img src="./images/card2.png" alt="tonic-snapshot" class="tonic-img">
+                <article class="card-section-text">
+                    <h2 class="heading-title">
+                        Tonic
+                    </h2>
+                    <article class="card-sub-container">
+                        <h2 class="title-canopy">CANOPY</h2>
+                        <img src="" alt="">
+                        <h2 class="title-developer">Back End Dev</h2>
+                        <img src="" alt="">
+                        <h2 class="title-year">2015</h2>
+                    </article>
```

Comment:
- [ ] Kindly make sure to add the dots which are acting as the separators between these contents.

   - ### YOUR DESIGN
  
       ![image](https://user-images.githubusercontent.com/87197412/212261756-ae5d6dfd-a5e4-4d21-8dd0-82de8584f075.png)

   - ### FIGMA DESIGN
   
      ![image](https://user-images.githubusercontent.com/87197412/212261945-f88e444c-3cd7-48cd-be0f-6e09795145ce.png)